src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ListViewTests.cs

```diff
@@ -4949,6 +4949,139 @@ public void ListView_VirtualMode_InvokeNotifyAboutLostFocus_RemoveListViewItemFr
+            ListViewItem listViewItem3 = new ListViewItem("3");
+            ListViewItem listViewItem4 = new ListViewItem("4");
+            ListViewItem listViewItem5 = new ListViewItem("5");
+            ListViewItem listViewItem6 = new ListViewItem("6");
+            ListViewItem listViewItem7 = new ListViewItem("7");
+            ListViewItem listViewItem8 = new ListViewItem("8");
+            ListViewItem listViewItem9 = new ListViewItem("9");
+
+            using ColumnHeader columnHeader1 = new System.Windows.Forms.ColumnHeader();
+            using ColumnHeader columnHeader2 = new System.Windows.Forms.ColumnHeader();
+
+            listView.Columns.AddRange(new System.Windows.Forms.ColumnHeader[] {
+            columnHeader1,
+            columnHeader2});
+            listView.HideSelection = false;
+            var listItems = new System.Windows.Forms.ListViewItem[] {
+            listViewItem1,
+            listViewItem2,
+            listViewItem3,
+            listViewItem4,
+            listViewItem5,
+            listViewItem6,
+            listViewItem7,
+            listViewItem8,
+            listViewItem9};
+            listView.Items.AddRange(listItems);
+            listView.View = System.Windows.Forms.View.SmallIcon;
+            listView.Size = new System.Drawing.Size(200, 200);
+
+            Func<string, ListViewItem> getItem = (itemText) => listItems.Single(item => item.Text == itemText);
+            Action<ListViewItem, SearchDirectionHint, string> testTheItem = (item, direction, resultText) =>
+             {
+                 if (resultText == null)
+                 {
+                     Assert.Null(item.FindNearestItem(direction));
+                 }
+                 else
+                 {
+                     Assert.Equal(getItem(resultText),item.FindNearestItem(direction) );
+                 }
+             };
+
+            var listViewItemToTest = getItem(item);
+            testTheItem(listViewItemToTest, SearchDirectionHint.Left, leftitem);
+            testTheItem(listViewItemToTest, SearchDirectionHint.Up, upitem);
+            testTheItem(listViewItemToTest, SearchDirectionHint.Right, rightitem);
+            testTheItem(listViewItemToTest, SearchDirectionHint.Down, downitem);
+        }
+        [WinFormsTheory]
+        [MemberData(nameof(ListView_FindNearestItem_Invoke_TestData))]
+        public void ListView_FindNearestItem_With_Images(string item, string leftitem, string upitem, string rightitem, string downitem)
+        {
+            using var imagecollection = new ImageList();
+            imagecollection.Images.Add(Bitmap.FromFile("bitmaps\\SmallA.bmp"));
+            imagecollection.Images.Add(Bitmap.FromFile("bitmaps\\SmallABlue.bmp"));
+
+            imagecollection.TransparentColor = System.Drawing.Color.Transparent;
+            imagecollection.Images.SetKeyName(0, "SmallA.bmp");
+            imagecollection.Images.SetKeyName(1, "SmallABlue.bmp");
```

Comment:
```suggestion
            using var imagecollection = new ImageList();
            imagecollection.Images.Add("SmallA.bmp", Form.DefaultIcon);
            imagecollection.TransparentColor = System.Drawing.Color.Transparent;
            imagecollection.Images.SetKeyName(0, "SmallA.bmp");
```
We can use a `Form.DefaultIcon` instead of an external file. The test will still check this case, but we will not need to add files